index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Subrat Dash!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">    
+</head>
+<body>
+    <header>
+        <h2>Welcome</h2>
+        <i class="fa fa-bars fa-2x"></i>

```

Comment:
- [ ] You are doing great so far. Please in this section as per the requirements; the hamburger icon should have a `nav` tag. The purpose for this is because is is to promote correct HTML semantics. the `nav` tag identifies an element as having some navigating functionality. In this case that is the Hamburger icon that will provide that functionality.
An implementation as simple as this will be very nice 👍: 
```suggestion
        <nav>
            <i class="fa fa-bars fa-2x"></i>
        </nav>

```